drivers/sensor/iis2iclx/iis2iclx.c

```diff
@@ -568,12 +564,12 @@ static int iis2iclx_init_chip(const struct device *dev)
 
 	k_usleep(100);
 
-	if (iis2iclx_accel_set_fs_raw(dev,
-				     IIS2ICLX_DEFAULT_ACCEL_FULLSCALE) < 0) {
+	LOG_INF("range is %d", fs);
```

Comment:
maybe `LOG_DBG`?

Reply:
ok, I was infact in doubt. Seems more for debug I agree.
Thanks